cli/deprecated_flags.go

```diff
@@ -26,6 +33,28 @@ func NewDeprecatedFlags(opts *options.TerragruntOptions) cli.Flags {
 				return nil
 			},
 		},
+		&cli.BoolFlag{
```

Comment:
Make sure to include strict controls for these. We should have them removed by 1.0.

Reply:
Done.